core/commands/object/object.go

```diff
@@ -293,7 +293,7 @@ var ObjectStatCmd = &cmds.Command{
 
 var ObjectPutCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Stores input as a DAG object, outputs its key.",
+		Tagline: "Store input as a DAG object, print its key.",
```

Comment:
I'm not sure. Print feels like it should mean stderr, to me, but I think this is just stdout. @whyrusleeping?